yamls/clusters/docker-cluster-one-3-control-plane-replicas.yaml

```
@@ -0,0 +1,35 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: "docker-cluster-one"
+  namespace: "default"
+spec:
+  clusterNetwork:
+    services:
+      cidrBlocks: ["10.128.0.0/12"]
+    pods:
+      cidrBlocks: ["192.168.0.0/16"]
+    serviceDomain: "cluster.local"
+  topology:
+    class: quick-start
+    version: v1.24.6
+    controlPlane:
+      replicas: 3
```

```suggestion
      replicas: 3   // Replicas changed from 1 to 3
```

Makes it easier for the user to understand what they are looking for. The diff that we are generating in the docs should help with this already but this will help add some more color.

---

Similar comment for `docker-cluster-one-second-worker-pool.yaml`

---

Good suggestion, thanks!